examples/time/virtual_time.rs

```diff
@@ -0,0 +1,230 @@
+//! Shows how `Time<Virtual>` can be used to pause, resume, slow down
+//! and speed up a game.
+
+use std::time::Duration;
+
+use bevy::{
+    input::common_conditions::input_just_pressed, prelude::*,
+    time::common_conditions::on_real_timer, window::WindowResolution,
+};
+
+fn main() {
+    App::new()
+        .insert_resource(ClearColor(Color::DARK_GRAY))
+        .add_plugins(DefaultPlugins.set(WindowPlugin {
```

Comment:
Do we really need to muck with the window settings here? If possible, we should leave this out to avoid distracting from the point of the example.

Reply:
I just wanted to prevent the sprites going offscreen, but I can leave the default (720p I think?) and hardcode the move amount.
Should I remove the BG/clear color too?

Reply:
Yeah, I would make all three of those changes.